medusa/providers/torrent/rss/rsstorrent.py

```diff
@@ -46,8 +46,9 @@ def __init__(self, name, url, cookies='',
         self.enable_daily = enable_daily
         self.enable_manualsearch = enable_manualsearch
         self.enable_backlog = enable_backlog
-        self.enable_cookies = True
+        self.enable_cookies = True if cookies else False
```

Comment:
Bool(cookies)

Reply:
Up 2 u

Reply:
Using bool() here is better indeed, changed.